src/storage/BaseProcessor.h

```diff
@@ -89,28 +90,24 @@ class BaseProcessor {
                        const std::string& start,
                        const std::string& end);
 
-    cpp2::ErrorCode to(kvstore::ResultCode code);
-
-    cpp2::ErrorCode writeResultTo(WriteResult code, bool isEdge);
+    nebula::cpp2::ErrorCode writeResultTo(WriteResult code, bool isEdge);
 
     nebula::meta::cpp2::ColumnDef columnDef(std::string name,
                                             nebula::meta::cpp2::PropertyType type);
 
-    void pushResultCode(cpp2::ErrorCode code, PartitionID partId);
+    void pushResultCode(nebula::cpp2::ErrorCode code, PartitionID partId);
 
-    void pushResultCode(cpp2::ErrorCode code, PartitionID partId, HostAddr leader);
+    void pushResultCode(nebula::cpp2::ErrorCode code, PartitionID partId, HostAddr leader);
 
-    void handleErrorCode(kvstore::ResultCode code, GraphSpaceID spaceId, PartitionID partId);
+    void handleErrorCode(nebula::cpp2::ErrorCode code,
```

Comment:
`pushResultCode` and `handleErrorCode` should only save one now, they are for sync method.

Reply:
`handleErrorCode `contains `handleLeaderChanged `and `pushResultCode`.
I think it is clearer to separate handleLeaderChanged and pushResultCode.
Of course they can also be merged together.